elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/MetaDataStore.java

```diff
@@ -186,6 +233,16 @@ public void addTable(Table table) {
         table.getArguments().forEach(arg -> addArgument(arg, version));
     }
 
+    /**
+     * Add a namespace metadata object.
+     *
+     * @param namespace Namespace metadata
+     */
+    public void addNamespace(Namespace namespace) {
+        // Add/Replicate Namespace for each version
+        hashMapDataStores.keySet().forEach(version -> addMetaData(namespace, version));
```

Comment:
Packages in Elide are versioned (see the ApiVersion) annotation.  Rather than copy the namespace for each version, I think we need to propagate the ApiVersion to the namespace as well.